alerts.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  alert_resource_group_name = "disposer-shared-${var.env}"
```

Comment:
The resource should be disposer-prod - https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/8999dec3-0104-4a27-94ee-6588559729d1/resourceGroups/disposer-prod/overview